deployment/src/main/java/io/quarkiverse/loggingmanager/deployment/LoggingManagerConfig.java

```diff
@@ -10,7 +10,7 @@
     /**
      * The base path, defaults to /loggers
```

Comment:
Let's take the opportunity to delete the ` defaults to /loggers` comment?

Reply:
Thanks I missed that.